avaframe/com1DFA/com1DFACfg.ini

```diff
@@ -0,0 +1,30 @@
+### Config File - This file contains the main settings for the simulation run
+## Copy to  local_com1DFACfg.ini and set you parameters
+# This file is part of Avaframe.
+
+
+[GENERAL]
+# Path to samos executable and AK_Attributes files
+samosAT = ./samosAT -files files/AK_Attributes/
+# Name of run cint file template
+RunCint = runBasic
+# Input format of release are files
+# -options: shp (shapfile), nxyz (list of polygon coordinates x,y,z=0))
+inputf = shp
+# True if full output shall be printed to terminal
+flagOut = False
+# True if entrainment areas shall be included
+flagEnt = True
+# True if resistance areas shall be included
+flagRes = True
+# True if parameter variation of MU option shall be included
+flagVarMU = False
+
+
+[AIMEC]
+# directory to store results for Aimec
+aimecDir = Outputs/com1DFA/
+# pressure limit for Aimec to perform computations
+calcPressureLimit = 10
+# width of the avalanche path domain for computations in Aimec
+domainWidth = 600
```

Comment:
I guess these will be moved to the aimec module?

Reply:
Not sure for the aimecDir = Outputs/com1DFA/ because you create the data in com1DFA and need to know where to export the data.
What is sure is that you don't need the calcPressureLimit and domainWidth any more and you don't need to create the aimecPathFile anymore. I think this will be done with aimec

Reply:
The aimec stuff will be moved to ana3AIMEC as a next step, so the com1DFA output is saved to Outputs/com1DFA and there will be a helper function that exports the required data from there.

Reply:
Shall we first move the things from Aimec or merge it as is and do the changes this in a new feature branch?

Reply:
Merge it as is and do the changes in a new branch...